plotly/figure_factory/_facet_grid.py

```diff
@@ -189,18 +190,22 @@ def _facet_grid_color_categorical(df, x, y, facet_row, facet_col, color_name,
     if not facet_row and not facet_col:
         color_groups = list(df.groupby(color_name))
         for group in color_groups:
-            trace = graph_objs.Scatter(
+            trace = dict(
                 x=group[1][x],
                 y=group[1][y],
```

Comment:
I'm having a hard time understanding what happens when the `histogram` mode sets both `x` and `y` here. Shouldn't only one of them be set?